tensorflow/python/kernel_tests/conv_ops_3d_test.py

```diff
@@ -220,13 +220,12 @@ def testConv3D1x1x1Filter(self):
         expected=expected_output)
 
   def testConv3D1x1x1Filter2x1x1Dilation(self):
-    if test.is_gpu_available(cuda_only=True):
```

Comment:
I don't think we can remove these GPU guards yet, since the normal CPU version still doesn't support it. There are two ways to workaround this:
1. Add a python guard for TF-MKL and modify the test guard to:
```
   if test.is_gpu_available(cuda_only=True) or test.is_tf_mkl_cpu:
```
or

2. Make the normal TF op support dilations as well. The Eigen implementation actually already supports dilations, just not the CPU TF ops.

Reply:
Solution 1 taken. Please have a look if you have time.